x-pack/qa/transport-client-tests/src/test/java/org/elasticsearch/xpack/enrich/client/EnrichTransportClientIT.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.enrich.client;
+
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.xpack.ESXPackSmokeClientTestCase;
+import org.elasticsearch.xpack.core.XPackClient;
+import org.elasticsearch.xpack.core.enrich.EnrichPolicy;
+import org.elasticsearch.xpack.core.enrich.action.DeleteEnrichPolicyAction;
+import org.elasticsearch.xpack.core.enrich.action.GetEnrichPolicyAction;
+import org.elasticsearch.xpack.core.enrich.action.PutEnrichPolicyAction;
+import org.elasticsearch.xpack.core.enrich.client.EnrichClient;
+import org.hamcrest.Matchers;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.nullValue;
+
+public class EnrichTransportClientIT extends ESXPackSmokeClientTestCase {
+
+    private static void assertEqualPolicies(EnrichPolicy expectedInstance, EnrichPolicy newInstance) {
+        assertThat(newInstance.getType(), Matchers.equalTo(expectedInstance.getType()));
```

Comment:
Maybe statically import this: `Matchers.equalTo`

and then remove: `import static org.hamcrest.CoreMatchers.equalTo;`